layers/water-mesh.js

```diff
@@ -409,5 +417,45 @@ export class WaterMesh extends BufferedMesh {
       // handle swimming action
       this.handleSwimAction(contactWater, localPlayer, WATER_HEIGHT);
     }
+    this.underWater = camera.position.y < WATER_HEIGHT;
+    this.material.uniforms.uTime.value = performance.now() / 1000;

```

Comment:
don't use `performance.now()`
you can get the `timestamp` or `timediff` from the `useFrame()` hook in the engine

Reply:
[use timeStamp](https://github.com/webaverse/heightfield/pull/19/commits/2374dbc10e8a5215fee65fefcf24fe8a254ddbfe)